kubetest2-gke/deployer/up.go

```diff
@@ -99,7 +99,10 @@ func (d *Deployer) CreateClusters() error {
 
 		r := retryCount
```

Comment:
why do we have this?

Reply:
also why is retryCount stored on the deployer?

Reply:
It's because we also need it for a few other functions, like `d.Up()`, `d.Kubeconfig()` and `d.EnsureFirewallRules()`.

Reply:
That line was added because the Go caveat - https://eli.thegreenplace.net/2019/go-internals-capturing-loop-variables-in-closures/. I have moved the code to a separate `tryCreateClusters()` function as per your suggestion, so that issue has gone.

Reply:
that's fair, it seems like the loop should directly increment d.retryCount still though?

I don't see why we have _three_ copies of this value. The capture one is reasonable.